ads/google/adsense.js

```diff
@@ -28,6 +44,9 @@ export function adsense(global, data) {
   validateData(data, [],
       ['adClient', 'adSlot', 'adHost', 'adtest', 'tagOrigin', 'experimentId',
         'ampSlotIndex', 'adChannel', 'autoFormat']);
+  if (data.autoFormat === 'rspv') {
```

Comment:
Actually, what do you think about putting this and the validateData call above in its own export?
That way we could implement the TODO in getAdUrl and not need to call it from two different places.